spec/feature_tests/user_stories_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'takeaway'
+
+describe "User stories" do
+
+let(:takeaway) {Takeaway.new}
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  describe 'User Story 1' do
+
+    it 'allows me to see a list of dishes with prices' do
+      expect{takeaway.show_dishes}.to output("starter : 5\nmains : 10\ndesert : 3").to_stdout
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)